new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/HelmParams.java

```diff
@@ -98,4 +98,5 @@ public String getFilter() {
   public String getChartVersion() {
     return chartVersion;
   }
+
```

Comment:
Is adding an extra line the only change of this class?

Reply:
I guess I added something and then removed it. yes looks like no changes in this class

Reply:
LGTM